mvp-spec/json-schemas/transmission/seed.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Seed",
+    "description": "The Seed gathers data related to the Addressable Content and sign them.",
+    "$id": "seed",
+    "type": "object",
+    "properties": {
+        "version": {
+            "$ref": "version.json"
+        },
+        "transaction_id": {
+            "$ref": "transaction_id.json"
+        },
+        "publisher": {
+            "type": "string",
+            "description": "The domain name of the Publisher that displays the Addressable Content",
+            "format": "hostname",
+            "examples": [
+                "publisher.com"
+            ]
+        },
+        "source": {
+            "description": "Signature based on input:\n```seed.source.domain + '\u2063' +\nseed.source.timestamp + '\u2063' +\nseed.transaction_id + '\u2063' +\nseed.publisher + '\u2063' +\ndata.identifiers[0].source.signature + '\u2063' +\ndata.identifiers[1].source.signature + '\u2063' +\n... + '\u2063' +\ndata.identifiers[n].source.signature + '\u2063' +\ndata.preferences.source.signature\n```",
```

Comment:
You need to double the backslashes in all the source description, otherwise it just displays as empty char.

Reply:
indeed. Thank you !